### PR TITLE
Set default DNS servers when using static VCH IP

### LINKED ISSUE #3060: Fixed IP VCH vm configuration does not honour the default DNS
version 0.7.0

When setting a static IP address to the VCH, when the flag `--dns-server` is not populated we set the DNS to the GW specified (Vs. 8.8.8.8 as the help suggests): 

```
--dns-server value                               DNS server for the client, external, and management networks. Defaults to 8.8.8.8 and 8.8.4.4 when not using DHCP 
```

Workaround: when setting a fixed IP explicitly use `--dns-server 8.8.8.8`

### PR DESCRIPTION
**Pull request by andrewtchin**
Fixes #3060 

```
root@chin-vic-10 [ ~ ]# cat /etc/resolv.conf 
nameserver 8.8.8.8
nameserver 8.8.4.4
options timeout:15
options attempts:5
```

### CODE REVIEW

File: cmd/vic-machine/create/create.go [hunk 1]
```diff
@@ -743,10 +743,16 @@ func (c *Create) processNetwork(network *data.NetworkConfig, netName, pgName, st
 
 // processDNSServers parses DNS servers used for client, external, mgmt networks
 func (c *Create) processDNSServers() error {
-	if len(c.dns) == 0 {
+	if c.Data.ClientNetwork.Empty() && c.Data.ExternalNetwork.Empty() && c.Data.ManagementNetwork.Empty() {
+		log.Warn("Specified DNS servers are ignored if static IP is not set on any networks. VCH will use DNS servers provided by DHCP.")
```
Comment (hmahmood) [inline cmd/vic-machine/create/create.go]: That is not the case. I fixed this about a week ago. Now you can specify dns server while not specifying a static ip.

**Comment by emlin** [comment]
lgtm 👍

**Comment by hmahmood** [comment]
lgtm

**Comment by andrewtchin** [comment]
@stuclem this requires a doc update. this is our behavior with the various options:
```
DHCP and no DNS specified -> DHCP provided nameservers
DHCP and DNS specified -> specified values
static and no DNS specified -> default to google DNS
static and DNS specified -> specified values
static and DHCP and DNS specified -> specified values
```

**Comment by andrewtchin** [comment]
♻️

**Comment by andrewtchin** [comment]
♻️

**Pull request merged**